languages/julia/exercises/concept/vehicle-purchase/.docs/introduction.md

```diff
@@ -0,0 +1,150 @@
+There are three primary conditional statements that are used in Julia:
+- `if`-statements
+- The ternary operator
+- Short-circuit evaluation
+
+## `if`-statements
+
+`if`-statements in Julia are similar to those seen in other languages:
+
+```julia
+julia> function ispositive(n)
+           if n > 0
+               println("n is positive!")
```

Comment:
What's with all the strings?

Reply:
Most are based on the swift implementation.

I was struggling to come up with return values that aren't booleans, because in that case you'd simply have `return n > 0` instead of an if statement. Returning or printing strings seemed like a way to avoid that. I don't mind changing these to something that makes more sense, though.

Reply:
I like that, changed it.